guides/common/modules/proc_using-novnc-to-access-virtual-machines.adoc

```diff
@@ -11,11 +11,16 @@ You can use your browser to access the VNC console of VMs created by {Project}.
 * Libvirt
 
 .Prerequisites
-* You must have a virtual machine created by {Project}.
+* You have a virtual machine created by {Project}.
 * For existing virtual machines, ensure that the *Display type* in the *Compute Resource* settings is *VNC*.
-* You must import the Katello root CA certificate into your {ProjectServer}.
+* You have imported the CA certificate used for {Project} into your browser.
 Adding a security exception in the browser is not enough for using noVNC.
-For more information, see {ConfiguringUserAuthenticationDocURL}importing-the-katello-root-ca-certificate[Installing the Katello root CA certificate] in _{ConfiguringUserAuthenticationDocTitle}_.
+ifdef::katello,orcharhino,satellite[]
+For more information, see {ConfiguringUserAuthenticationDocURL}importing-the-katello-root-ca-certificate[Importing the Katello root CA certificate] in _{ConfiguringUserAuthenticationDocTitle}_.
+endif::[]
+ifdef::foreman-el,foreman-deb[]
+To get the path on your {ProjectServer}, search for *Websockets SSL certificate* in your {Project} settings.
+endif::[]
```

Comment:
I'm tempted to leave leave this out altogether because I don't think it really helps people do the right thing. If we want to get this right, we should fix up the `importing-the-katello-root-ca-certificate` procedure.
```suggestion
```

Reply:
Question: Do you think a publicly accessible template similar to `app/views/unattended/provisioning_templates/registration/foreman_ca_refresh.erb` could work to get the correct cert from Foreman?

Reply:
https://github.com/theforeman/foreman-documentation/issues/4073